contest/urls.py

```diff
@@ -22,6 +22,10 @@
 from contest import views
 
 urlpatterns = patterns('',
-    url(r'^$',views.index,name='index'),
+    url(r'^$',views.archive,name='archive'),
+    url(r'^new/$',views.new,name='new'),
+    url(r'^edit/(?P<contest_id>\d+)/$',views.edit,name='edit'),
+    url(r'^delete/(?P<contest_id>\d+)/$',views.delete,name='delete'),
     url(r'^(?P<contest_id>\d+)/$',views.contest,name='contest'),
-    )
+    url(r'^register/(?P<contest_id>\d+)/$',views.register,name='register'),
```

Comment:
We need a button and view for registration, but do we need a page for it?
A user can register a contest by pressing a button, but is it appropriate for users to register by entering an url?